src/index.scss

```diff
@@ -1 +1,10 @@
-@import 'assets/styles/reset.scss';
\ No newline at end of file
+@import '~assets/styles/reset.scss';
+@import '~assets/styles/colors.scss';
+
+* {
+  font-family: 'Roboto', sans-serif !important;
+}
+
+html, body {
+  background: $aqua-spring;
```

Comment:
`Suggestion`

I like to keep one file for all the `SCSS` variables I'll use in my project, and in that same file,
specify colors not by their specific color, but by its usage. 

For example:

```SCSS
/******** Colors **********/

$aqua-spring: #effafa;
$white: #fff;
$black: #000;

$background-color: $aqua-spring;
$text-color: $black;
$muted-text-color: mix($black, #fff, 5%);

// In `index.scss`

html, body {
  background: $background-color;
}
```

It may look repetitive, but I think it adds maintainability. 

[Minimal Mistakes] has the best `SCSS` architecture I've ever seen. Check the [_variables.scss] file.

[Minimal Mistakes]: https://github.com/mmistakes/minimal-mistakes/tree/master/_sass/minimal-mistakes
[_variables.scss]: https://github.com/mmistakes/minimal-mistakes/blob/master/_sass/minimal-mistakes/_variables.scss

Reply:
Sounds good to me, corrected